src/languages/arcade.js

```diff
@@ -75,7 +249,8 @@ export default function(hljs) {
   ]);
 
   return {
-    name: 'ArcGIS Arcade',
+    aliases: [ 'arcade' ],
+    case_insensitive: true,
```

Comment:
This quite possibly could be why your relevance broke (if this is the language you were referring to)... that changes a lot of things (makes them less precise, more likely to false positive, etc).